From cf8730b5c75bf1ea8078cbf52f45549b3c1e2aec Mon Sep 17 00:00:00 2001
From: tonishi92 <tatsuo.onishi@gmail.com>
Date: Tue, 17 Jan 2023 11:01:52 +0100
Subject: [PATCH 04/54] Fix chem_driver init debug message

In chem/chem_driver.F remove a confusing wrf_debug message implying that the
chem_opt option used is unknown, when in reality this is triggered by
tracer_opt=0.
---
 chem/chem_driver.F | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/chem/chem_driver.F b/chem/chem_driver.F
index 8650b944..cd39eeaf 100755
--- a/chem/chem_driver.F
+++ b/chem/chem_driver.F
@@ -664,7 +664,11 @@
     CASE (TRACER_TEST3)
        CALL wrf_debug(15,'tracer mode: 10 tracers')
      CASE DEFAULT
-       CALL wrf_debug(15,'calling chem_opt=? from chem_driver')
+       if(config_flags%tracer_opt > 0 )then
+       CALL wrf_debug(15,'calling tracer_opt=? from chem_driver')
+       else
+       CALL wrf_debug(15,'not calling tracer transport in chem_driver')
+       endif
    END SELECT tracer_select
 
 ! initialize cam_mam local arrays
-- 
2.31.1

